tests/test_client.py

```diff
@@ -5,6 +5,11 @@
 from swiftype_app_search.exceptions import InvalidDocument
 
 
+def match_request_body(body_text):
```

Comment:
you should be able to remove this function now that you're using a lambda

Reply:
Ah, yeah, meant to do that. Thanks for pointing that out.